### DISCUSSION TITLE
Welcome to reduced.to Discussions!

### DISCUSSION THREAD

**Discussion by origranot** [origranot/reduced.to#121] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by GuillemPM** [comment]
I will begin here 💯 

Hey all! 
I'm a full-stack developer currently working on creating and maintaining VSCode extensions for my company.
My main stack is MEVN, maybe MEQN soon? 😄
Enthusiast of **Qwik** since I started contributing to this repo.
Things I contributed to URL-Shortener so far:
* Migrate backend to NestJS
* Migrate frontend to Qwik
* Make the project deployable as SSG in a single app
* Currently working on Migrate UI from Bootstrap to Tailwindcss (Supercharged with DaisyUI)

I hope to read your comments and contributions below 🔥🚀

**Comment by Ali-Amechghal** [comment]
Hey,

I'm wondering, what is the real reason for using NestJs for a small project like this one ? for someone coming from Java/Spring and familiar with DI , IMHO  , i think this is overkilling , i would like to know what you think about that

Thank you,

**Reply by origranot** [reply]
Hey @Ali-Amechghal,

Thank you for your opinion, we are just started 😄 
We are planning to add authentication and much more features upon this.
I personally really like NestJS, I truly believe it's important to have a strong fundamentals 💯

**Reply by Ali-Amechghal** [reply]
Hey @origranot ,

Thank you for your answer ^_^,
I'm not a big fun of NestJs and i think the modular architecture can only be usefull in entreprise projects where we need to implement DDD, CQRS or other architecture patterns, please let me know what you think about that,

Thank you :)

**Reply by origranot** [reply]
Hey @Ali-Amechghal, 
I am not totally agree with you, I think that CQRS or other event sourcing architecture is for robust systems with a lot of requests.
I think that NestJS is useful despite the size of the project. It super simple, easy to understand and maintainable.
There are a lot of great built in features in NestJS, not only the modular architecture and the DI. (that are awesome).
